js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1167,62 +1240,83 @@ const EditingController = modules.ViewController.inherit((function() {
         },
 
         editCell: function(rowIndex, columnIndex) {
+            return this._editCell({ rowIndex, columnIndex });
+        },
+
+        _editCell: function(options) {
             const d = new Deferred();
             let coreResult;
 
             this.executeOperation(d, () => {
-                coreResult = this._editCellCore(rowIndex, columnIndex);
+                coreResult = this._editCellCore(options);
                 when(coreResult)
                     .done(d.resolve)
                     .fail(d.reject);
             });
 
             return coreResult !== undefined ? coreResult : d.promise();
         },
-        _editCellCore: function(rowIndex, columnIndex) {
-            const that = this;
-            const columnsController = that._columnsController;
-            const dataController = that._dataController;
-            const items = dataController.items();
-            const item = items[rowIndex];
-            const params = {
-                data: item && item.data,
-                cancel: false
-            };
-            const oldEditRowIndex = that._getVisibleEditRowIndex();
+
+        _normalizeEditCellOptions: function({ oldColumnIndex, oldRowIndex, columnIndex, rowIndex }) {
```

Comment:
Since we expect that the method returns something, I think it is better to name it **_getNormalizedEditCellOptions**.
```suggestion

        _getNormalizedEditCellOptions: function({ oldColumnIndex, oldRowIndex, columnIndex, rowIndex }) {
```